lib/feature_requests.rb

```diff
@@ -0,0 +1,27 @@
+module Lita
+  # Returns cards that have been in Review column for more than two days
+  class FeatureRequests
+
+    SEVEN_DAYS = (60*60*168)
```

Comment:
I had to stop and think about what 168 was. You should just put all the steps in there (e.g. `60*60*24*7`).

That way if we want to change it to 6 days, or 10 days it's a no-brainer.